assets/js/wfparameters/ColumnSelector.js

```diff
@@ -82,22 +90,25 @@ export default class ColumnSelector extends React.PureComponent {
             title='Select None'
             onClick={this.onClickSelectNone}
             className='mc-select-none content-4 t-d-gray'
-            >
+          >
             None
           </button>
         </div>
-        <Select
-          isMulti
-          name='columns'
-          options={columnOptions}
-          menuPortalTarget={document.body}
-          className='react-select'
-          classNamePrefix='react-select'
-          onChange={this.onSelectColumn}
-          value={selectedColumns}
-          isClearable={false}
-          placeholder='Select columns'
-        />
+        <div className="column-multi-select">
```

Comment:
`multi-select-input-group` would be a better class name. ("Input group" is Bootstrap's name for inputs and buttons that render together.)